packages/react-microlink/src/index.js

```diff
@@ -13,17 +13,17 @@ class Microlink extends Component {
   }
 
   componentDidMount () {
-    if (this.props.data) return this.setData({data: this.props.data})
-    return this.fetchData().then(this.setData)
+    if (this.props.data) return this.setData(this.props.data)
+    return this.fetchData().then(({data}) => this.setData(data))
```

Comment:
If `setData` will receive an object, you don't need setup `then`.

Also the method could be write in one line

```jsx
return !this.props.data ? this.fetchData().then(this.setData) : this.setData(this.props.data)
```

I negate `this.props.data` because `fetchData` is the most common action